prime-router/settings/STLTs/MA/ma-phd.yml

```diff
@@ -0,0 +1,108 @@
+# To submit this to localhost, run:
+#
+# ./prime multiple-settings set --input ./setting/staging/ma-phd.yml
+#
+---
+- name: "ma-phd"
+  description: "Massachusetts Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "MA"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "ma-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/MA/MA-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "MA-MDPH"
+        receivingApplicationOID: "2.16.840.1.113883.19.3.2"
+        receivingFacilityName: "MA-MDPH"
+        receivingFacilityOID: "2.16.840.1.113883.19.3.2"
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB:
```

Comment:
Do we keep these settings rather than move them into the receiver transforms?

Reply:
No, we don't need to.  Thanks.